modules/claims_api/config/locales/en.yml

```diff
@@ -0,0 +1,4 @@
+en: 
+  date:
+    formats:
+      va_form: "%m/%d/%Y"
```

Comment:
out of curiosity, what's using this?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/blob/insert_2122_signatures/modules/claims_api/lib/claims_api/power_of_attorney_pdf_constructor.rb#L37